src/authentication.ts

```diff
@@ -25,13 +26,16 @@ export const authenticateRequest = async (token: string, tenantId: string) => {
 
 export const authenticateLambda = async (
   headers: APIGatewayProxyEventHeaders,
-  tenantId: string
+  tenantId: string,
+  logger: LambdaLogger<any>
 ) => {
   if (!headers.authorization) {
     return [null, "No authorization headers"] as const;
   }
   try {
+    logger.info("ENGEXPUTILS013");
```

Comment:
`logger.debug` please, we only need these if we're debugging something (like how long it takes to log in)

Reply:
changed